integrations-and-supported-tools/mosaicml-composer/notebooks/Neptune_MosaicML_Composer.ipynb

```diff
@@ -0,0 +1,372 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "EiPydbIreluR"
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune as an anonymous user, with zero setup.\n",
+    "\n",
+    "If you want to see the example logged to your own workspace instead:\n",
+    "\n",
+    "  1. Create a Neptune account. [Register &rarr;](https://neptune.ai/register)\n",
+    "  1. Create a Neptune project that you will use for tracking metadata. For instructions, see [Creating a project](https://docs.neptune.ai/setup/creating_project) in the Neptune docs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "NTVvWzR-eluS"
+   },
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -U neptune torch torchvision #mosaicml #TODO: Uncomment after release\n",
+    "\n",
+    "# TODO: Remove after release\n",
+    "%pip install git+https://github.com/AleksanderWWW/composer.git@neptune-logger"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torchvision import datasets, transforms\n",
+    "from composer import Trainer\n",
+    "from composer.models import mnist_model\n",
+    "from composer.loggers import NeptuneLogger\n",
+    "from composer.algorithms import LabelSmoothing, BlurPool, ProgressiveResizing"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare dataset and dataloaders"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transform = transforms.Compose([transforms.ToTensor()])\n",
+    "train_dataset = datasets.MNIST(\"data\", download=True, train=True, transform=transform)\n",
+    "eval_dataset = datasets.MNIST(\"data\", download=True, train=False, transform=transform)\n",
+    "train_dataloader = DataLoader(train_dataset, batch_size=128)\n",
+    "eval_dataloader = DataLoader(eval_dataset, batch_size=128)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## (Neptune) Create `neptune_logger`\n",
+    "\n",
+    "To create a new run for tracking the metadata, you tell Neptune who you are (`api_token`) and where to send the data (`project`).\n",
+    "\n",
+    "You can use the default code cell below to create an anonymous run in the public project [common/mosaicml](https://app.neptune.ai/o/common/org/mosaicml). **Note**: Public projects are cleaned regularly, so anonymous runs are only stored temporarily.\n",
```

Comment:
Sorry to nitpick but IMO we should use the library rather than the org name here (`common/composer` or `common/mosaicml-composer`). It would be similar to Hugging Face vs Transformers, no?